conf/app.routes

```diff
@@ -79,7 +82,8 @@ POST       /register/registration-type                   controllers.WhatAreYouR
 GET        /register/change-registration-type            controllers.WhatAreYouRegisteringAsController.onPageLoad(mode: models.Mode = models.CheckMode)
 POST       /register/change-registration-type            controllers.WhatAreYouRegisteringAsController.onSubmit(mode: models.Mode = models.CheckMode)
 
-GET        /register/could-not-confirm-identity          controllers.WeCouldNotConfirmController.onPageLoad(mode: models.Mode = models.CheckMode)
+GET        /register/could-not-confirm-identity          controllers.WeCouldNotConfirmController.onPageLoad(key: String  = "identity")
+GET        /register/could-not-confirm-business          controllers.WeCouldNotConfirmController.onPageLoad(key: String  = "business")
```

Comment:
@iservport Is it working, URL could be something like 
`/register/could-not-confirm/:identity controllers.WeCouldNotConfirmController.onPageLoad(identity: String)`